bridge/bridgeapp/templates/home.html

```diff
@@ -21,6 +21,13 @@ <h2>Spanning Questions And Answers</h2>
                     </a></li>
                     {% endfor %}
                 </ul>
+                {% if user.is_authenticated %}
+                <h3>You are logged in as: {{ user.username }} </h3>
+                <h4>Click here to <a href="{% url 'logout' %}">log out</a></h4>
+                {% else %} 
+                Click here to <a href="{% url 'login' %}">login</a> / 
+                New user? Click here to <a href="{% url 'register' %}">register</a> 
+                {% endif %} 
             </nav>
```

Comment:
The messages appended along with the anchor links to the end of the site <nav> is redundant. The user know whether s/he is logged in and authorized to use the site by the mere fact that s/he can see the home page.

The login/sign-up page is separate and you just added requirement for authenticatino to access the home page per  commit #db65c40f1e6abb3929751164dfee7f1b8dcaac30.

A log-out button appearing on the navbar is more than sufficient to tell the user what its function is; the whole "Click here ..." is superfluous.

The content starting with "Click here to <a href="{% url 'login' %} ...." is rendered utterly redundant and inappropriate by the fact that the user can only see them when logged in.

Please reconsider and revise.